tests/waku_sync/test_protocol.nim

```diff
@@ -139,3 +140,73 @@ suite "Waku Sync - Protocol Tests":
         assert hashes.isOk(), $hashes.error
         check:
           hashes.value.len == 0
+
+
+    asyncTest "sync 3 nodes cyclic":
```

Comment:
Multiparty sync is planned for later. Simple client <-> server is what we need to tests for now.

As for tests in general we may want to test with more messages ~10-100K with 1, 10, 100, 1K, 10K differences.

I'm not sure I understand what are we testing here?

Here's a couple of ideas;
empty client <-> empty server
empty client <-> full server
full client <-> empty server
100K with 1 random diff + simple compute time measure 
100K with 10K random diffs  + simple compute time measure
etc...

Reply:
Ok, just wanted to test 3 nodes syncing with each other, wanted to verify same node working as sync client and server.
Thanks for the input, my idea is to have tests similar to what you had mentioned above as next step:)

Reply:
Per our 1:1, I now understand!